doc/classes/IP.xml

```diff
@@ -79,6 +88,17 @@
 				Returns a given hostname's IPv4 or IPv6 address when resolved (blocking-type method). The address type returned depends on the [enum Type] constant given as [code]ip_type[/code].
 			</description>
 		</method>
+		<method name="resolve_hostname_addresses">
+			<return type="Array">
+			</return>
+			<argument index="0" name="host" type="String">
+			</argument>
+			<argument index="1" name="ip_type" type="int" enum="IP.Type" default="3">
+			</argument>
+			<description>
+                Resolves a given hostname in a blocking way. Addresses are returned as an [Array] of IPv4 or IPv6 addresses depending on [code]ip_type[/code].
```

Comment:
There's a style issue here, the indentation should be done with tabs, not spaces.

Reply:
Whoops, my bad!